packages/edit-site/src/components/save-button/index.js

```diff
@@ -38,10 +39,15 @@ export default function SaveButton( {
 	const disabled = isSaving || ! activateSaveEnabled;
 
 	const getLabel = () => {
+		if ( isSaving ) {
+			return __( 'Saving' );
+		}
 		if ( disabled ) {
 			return __( 'Saved' );
 		}
 
+		if ( defaultLabel ) return defaultLabel;
```

Comment:
Not a huge deal since `SaveButton` isn't a component that we export to third parties but I think the semantics around `defaultLabel` are a little confusing. (When will it be shown?) If possible I think it'd be better to have a `label` prop that allows callers to override the label altogether. That way it's clearer that when `label` is set then that's what will appear.

Reply:
@noisysocks main reason I called it `defaultLabel` was that the button label changes depending on saving state. If we called it label we'd have to also provide labels for each state (e.g. disabled, saving etc)